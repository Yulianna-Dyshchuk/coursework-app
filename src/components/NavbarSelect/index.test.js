import React from "react";
import { shallow } from "enzyme";
import { NavbarSelect } from "./index";

describe("NavbarSelect component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<NavbarSelect />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props", () => {
    const component = shallow(<NavbarSelect links={[]} baseUrl="/" />);

    expect(component).toMatchSnapshot();
  });
});
