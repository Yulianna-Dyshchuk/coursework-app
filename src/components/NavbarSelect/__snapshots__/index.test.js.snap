// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarSelect component should render correctly with props 1`] = `
<div>
  <Link
    to="/portfolio"
  >
    <WithStyles(ForwardRef(Typography))
      variant="subtitle1"
    >
      Portfolio
    </WithStyles(ForwardRef(Typography))>
  </Link>
</div>
`;

exports[`NavbarSelect component should render correctly without props 1`] = `
<div>
  <Link
    to="/portfolio"
  >
    <WithStyles(ForwardRef(Typography))
      variant="subtitle1"
    >
      Portfolio
    </WithStyles(ForwardRef(Typography))>
  </Link>
</div>
`;
